www/result.inc

```diff
@@ -143,6 +143,8 @@ $page_description = "Website performance test result$testLabel.";
                     $fvMedian = $testResults->getMedianRunNumber($median_metric, false);
                     $rvMedian = $testResults->getMedianRunNumber($median_metric, true);
 
+                    echo "<a href='/jsonResult.php?test=$id'>View JSON result</a><br>";

```

Comment:
Could you add a pretty=1 query param to get the pretty-printed version of the JSON result?

Reply:
Sure, of course!

40b33df354689725e78a198215f41c41a024d825